### PR TITLE
feat(suite): implement screento notify user about wallet remember

### PR DESCRIPTION
**Pull request by peter-sanderson**
Upgrades the flow of the Wallet Remember (now called View Only)

-----

- [ ] Rename Remember Wallet in code to something like  ViewOnlyWallet

### CODE REVIEW

File: packages/components/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -85,19 +86,23 @@ const CardComponent = forwardRef<HTMLDivElement, CardProps & { paddingType: Padd
...
 
-        return label ? (
-            <Container>
-                <LabelContainer $paddingType={paddingType}>{label}</LabelContainer>
-                <CardComponent {...props} ref={ref} />
-            </Container>
-        ) : (
-            <CardComponent {...props} ref={ref} />
+        return (
+            <ComponentFrame margin={margin} maxWidth={maxWidth}>
```
Comment (peter-sanderson) [inline packages/components/src/components/Card/Card.tsx]: Todo: check it this is the cause of the bad card size (not 100%)

**Comment by peter-sanderson** [comment]
Handed over to @jvaclavik

File: packages/components/src/components/Tooltip/TooltipFloatingUi.tsx [hunk 2]
```diff
@@ -78,7 +80,7 @@ export const useTooltip = ({
         open,
         onOpenChange: setOpen,
         whileElementsMounted: autoUpdate,
-        middleware: [offset(offsetValue), flip(), shift(), arrow({ element: arrowRef })],
+        middleware: [offset(offsetValue), flip(), shiftFUI(shift), arrow({ element: arrowRef })],
```
Comment (peter-sanderson) [inline packages/components/src/components/Tooltip/TooltipFloatingUi.tsx]: What FUI means?

File: packages/suite/src/views/view-only/ViewOnlyTooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    gap: ${spacingsPx.xs};
+`;
+
+const TextContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${spacingsPx.xxs};
+`;
+
+// Todo: Translations
```
Comment (peter-sanderson) [inline packages/suite/src/views/view-only/ViewOnlyTooltip.tsx]: :point_up:

**Review by peter-sanderson — COMMENTED** [review]
Approve (cannot approve my own PR)

Just please fix the translations.

File: packages/components/src/components/Tooltip/TooltipFloatingUi.tsx (hunk 2 above)
Comment (jvaclavik) [reply packages/components/src/components/Tooltip/TooltipFloatingUi.tsx]: Floating UI :D

**Review by jvaclavik — APPROVED** [review]

**Pull request merged**